com.unity.render-pipelines.core/Documentation~/Common/light-anchor.md

```diff
@@ -0,0 +1,16 @@
+# Light Anchor
+
+![](Images/LightAnchor0.png)
+
+A Light Anchor must be conncted to a Light. Which allow us to light a scene on a Rendered Camera Space.
+
+## Properties
+
+| **Property**    | **Description**                                              |
+| --------------- | ------------------------------------------------------------ |
+| Orbit | Control the Orbit of the light, can be controled with the first green knob. |
+| Evevation | Control the Elevation of the light, can be controled with the second blue knob. |
+| Roll | Control the roll, usefull if the light had an IES or a Cookie, can be controled with the third gray knob. |
+| Distance | Controls how far 'back', the light is placed from its anchor. |
+| Up Direction | Specifies the space in which the up direction of the anchor is defined. Local is relative to the camera. |
+| Common | Common light preset: Rim Left, Kick Left, Bounce Left, Fill Left, Hair, Fill Right, Bounce Right, Kick Right, Rim Right. |
```

Comment:
I don't think we need to list all the options of this property in a description. Especially if we're not explaining what they do.
I suggest the following, let me know if I've misunderstood how this operates: 

Assigns a preset to the light component based on the behaviour of studio lights.